src/SignalR/clients/ts/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn expects there to be a lock file even it if is empty. Commiting this to avoid the annoyance of always having this as an untracked file.